pyntc/devices/asa_device.py

```diff
@@ -462,6 +506,17 @@ def file_copy(
 
     # TODO: Make this an internal method since exposing file_copy should be sufficient
     def file_copy_remote_exists(self, src, dest=None, file_system=None):
+        """
+        Copy``src`` file to device.
+
+        Args:
+            src (str): [description]
+            dest (str, optional): [description]. Defaults to None.
+            file_system (str, optional): [description]. Defaults to None.
+
+        Returns:
+            bool: True if the file exists on the device and the md5 hashes match. Otherwise, false.
```

Comment:
For major functions like this one, I would try to add an example of how this would be used in command line. See: https://github.com/networktocode/pyntc/blob/main/pyntc/devices/aireos_device.py#L53-L58

Reply:
I would consider `major` function one that is used a lot, has input arguments and returns something.

Reply:
Sounds good. I will keep this in mind as I continue to add docstrings. Thanks!